Gems/WarehouseAutomation/README.md

```diff
@@ -0,0 +1,47 @@
+[![Apache License, Version 2.0][apache_shield]][apache]
+
+# Warehouse Automation Gem for Open 3D Engine (O3DE)
+
+## Requirements
+- Any O3DE project with the [ROS 2 Gem](https://github.com/o3de/o3de-extras/tree/development/Gems/ROS2) enabled.
+
+Please refer to [O3DE documentation](https://docs.o3de.org/docs/user-guide/gems/) to learn more about Gems and about registering Gems in the system and O3DE projects.
+
+## Description
+This Gem contains a set of models and prefabs with underlying O3DE components that can be used to create a warehouse project suitable for robotic simulations. In particular, it delivers a conveyor belt simulation and a proximity sensor that can be used to control the belt. E.g., it can be used to stop to belt when transported goods reach a certain position on a belt. The conveyor belt is fragmented into smaller pieces which can be put together in any configuration. Additionally, this Gem delivers an asset with a sample palletizer configuration, in which one conveyor belt splits into two pick-up stations for robotic arms and an extra roller conveyor for putting aside goods.
+
+The following assets are available in the Gem as O3DE prefabs:
+- `./Assets/Factory/ConveyorLine.prefab` - a sample palletizer configuration
+- `./Assets/Factory/ConveyorElements/ConveyorSegments_Turnout.prefab` - conveyor belt's element that splits into two and allows for goods distribution using a mechanized flap
+- `./Assets/Factory/ConveyorElements/ConveyorSegments_Straight.prefab` - conveyor belt's straight element
+- `./Assets/Factory/ConveyorElements/ConveyorSegments_Turn_90.prefab` - conveyor belt's turn element (90 degrees)
+- `./Assets/Factory/ConveyorElements/ConveyorSegments_End.prefab` - conveyor belt's end element with an extra flap to keep transported goods positioned
+
+Images of all prefabs are given in the next section.
+
+The proximity sensor is an O3DE component with no corresponding visualization. It uses `AzPhysics::RayCastRequest` to check for collisions in its range and returns continuously the result of this test via `ProximitySensorNotificationBus`. Any O3DE component can connect to this bus and react to certain events. The sensor and the bus are also available for scripting in [ScriptCanvas](https://docs.o3de.org/docs/user-guide/scripting/script-canvas/). It is possible to set the frequency and the range of the sensor. The visualization beam can be disabled.
+
+The Gem was implemented primarily for [ROSCon2023Demo](https://github.com/RobotecAI/ROSCon2023Demo) project and it was separated for easier reuse in other works. More details about robotic simulations can be found in [ROS 2 Gem documentation](https://docs.o3de.org/docs/user-guide/interactivity/robotics/). 
```

Comment:
```suggestion
More details about robotic simulations can be found in [ROS 2 Gem documentation](https://docs.o3de.org/docs/user-guide/interactivity/robotics/). 
```

Reply:
Note that the Gem is also used in the Manipulation Project Template.

Reply:
I decided to keep the information about the `ROSCon2023Demo` and added the `ManipulationTemplate`. It is an added value in readme when it directs to the real use case of the Gem.